sockeye/train.py

```diff
@@ -751,6 +796,14 @@ def train(args: argparse.Namespace) -> training.TrainState:
 
         training_model = model.SockeyeModel(model_config)
 
+        # Stopping criteria
```

Comment:
would this be better-placed inside `check_arg_compatibility`?

Reply:
Good idea